src/OctoshiftCLI.IntegrationTests/GhesToGithub.cs

```diff
@@ -83,7 +87,7 @@ await retryPolicy.Retry(async () =>
         });
 
         await _targetHelper.RunGeiCliMigration(
-            $"generate-script --github-source-org {githubSourceOrg} --github-target-org {githubTargetOrg} --ghes-api-url {GHES_API_URL} --download-migration-logs", _tokens);
+            $"generate-script --github-source-org {githubSourceOrg} --github-target-org {githubTargetOrg} --ghes-api-url {GHES_API_URL} --use-github-storage {useGithubStorage} --download-migration-logs", _tokens);
```

Comment:
In case of `false` for `useGithubStorge` it shouldn't add the option at all.

Reply:
Removing int test logic for now